index.html

```diff
@@ -3,96 +3,153 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="./css/style.css" />
+    <meta property="og:image" content="./image/logo.svg" />
+    <meta property="og:url" content="https://linkbrary-kwan.netlify.app/" />
+    <meta
+      property="og:description"
+      content="세상의 모든 정보를 쉽게 저장하고 관리해보세요."
+    />
+    <meta property="og:title" content="Linkbrary" />
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:image" content="./image/logo.svg" />
+    <meta
+      name="twitter:description"
+      content="세상의 모든 정보를 쉽게 저장하고 관리해보세요."
+    />
+    <meta name="twitter:title" content="Linkbrary" />
+    <link rel="stylesheet" href="./css/reset.css" />
     <link rel="stylesheet" href="./css/_variables.css" />
+    <link rel="stylesheet" href="./css/style.css" />
     <script
       src="https://kit.fontawesome.com/35dfed9802.js"
       crossorigin="anonymous"
     ></script>
     <title>Linkbrary</title>
   </head>
   <body>
-    <nav>
-      <a class="home__button" href="./index.html"
-        ><img src="./image/logo.svg"
+    <header class="nav-container">
+      <a class="home-button" href="./index.html"
+        ><img class="home-button__img" src="./image/logo.svg"
       /></a>
-      <a class="sign__button" href="./pages/signin/index.html">로그인</a>
-    </nav>
+      <a class="login-button" href="./pages/signin/index.html">로그인</a>
+    </header>
 
-    <header>
-      <div class="header__content">
-        <h1>
-          <span>세상의 모든 정보</span>를<br />쉽게 저장하고 관리해 보세요
+    <div class="hero-container">
+      <div class="hero-content">
+        <h1 class="hero-content__title">
+          <span class="hero-content__highlight">세상의 모든 정보</span
+          >를<br />쉽게 저장하고 관리해 보세요
         </h1>
-        <a href="./pages/signup/index.html">링크 추가하기</a>
-        <img src="./image/main_img.svg" />
+        <a class="hero-content__cta-btn" href="./pages/signup/index.html"
+          >링크 추가하기</a
+        >
+        <img
+          viewBox="0 0 100 100"
+          class="hero-content__img"
+          src="./image/main_img.svg"
+        />
       </div>
-    </header>
+    </div>
 
-    <main>
-      <div class="main__card">
-        <div class="main__card--text">
-          <h2>
-            <span id="special-text-first">원하는 링크</span>를<br />저장하세요
+    <main class="main-container">
+      <div class="main-card">
+        <div class="main-card__text-box">
+          <h2 class="main-card__title">
+            <span id="special-text-first">원하는 링크</span>를 저장하세요
           </h2>
-          <p>
+          <p class="main-card__text">
```

Comment:
@kuum97 

```suggestion
          <p class="main-card__description">
```

보통 컨벤션이, 타이틀과 디스크립션으로 구분됩니다.
이를 맞춰 디스크립션으로 적용하는게 더 올바른 네이밍 같아보여요!